tests/server/api/v1/test_questions.py

```diff
@@ -38,11 +39,33 @@
 
 
 @pytest.mark.parametrize(
-    "QuestionFactory, expected_settings",
+    "QuestionFactory, payload, expected_settings",
     [
-        (TextQuestionFactory, {"type": "text", "use_markdown": False}),
+        (
+            TextQuestionFactory,
+            {"title": "New Title", "description": "New Description", "settings": {"use_markdown": True}},
```

Comment:
I would say to be more explicit on with question setting type consumers are trying to update. I miss the `settings.type` value as a required one.